ethereum/spec/src/main/java/tech/pegasys/teku/spec/datastructures/builder/BlobsBundle.java

```diff
@@ -22,6 +22,7 @@
 import tech.pegasys.teku.spec.datastructures.type.SszKZGCommitment;
 import tech.pegasys.teku.spec.datastructures.type.SszKZGProof;
 
+// TODO: FULU with cell proofs
```

Comment:
My understanding is that the only meaningful change in this object was the now strict length of all list fields? What is this TODO about?

Reply:
Also, do we need to update `BlobsBundleSchema` with the update size for the proofs (`FIELD_ELEMENTS_PER_EXT_BLOB * MAX_BLOB_COMMITMENTS_PER_BLOCK`)?

Reply:
we don't have builder specs (or maybe I've not seen it yet), not sure
I don't want to change BlobsBundleSchema, I've separate BlobsCellBundleSchema